Automattic/newspack-plugin
# fix(ras): email to login conversion

### All Submissions:






### Changes proposed in this Pull Request:



Without strict user email sanitization (the second argument to `sanitize_user`), the WC customer creation would fail on certain characters.

Incorporates changes from #2980.

### How to test the changes in this Pull Request:

1. Ensure WooCommerce is active and RAS set up
1. On `release`, try to register using the Registration block with an email that contains a `'` character
2. Observe it fails
3. Try again on this branch, observe the registration goes as expected

### Other information:
